src/v3/src/transformer/layout/recovery/transformIdentityRecovery.ts

```diff
@@ -38,9 +44,19 @@ export const transformIdentityRecovery: IdxStepTransformer = ({ formBag, widgetP
     uischema.elements,
   ) as FieldElement;
 
+  const submitBtnElement: ButtonElement = {
```

Comment:
the submit button needed to be manually added to this transformer to be consistent with the identify flow